geode-serialization/src/main/java/org/apache/geode/internal/serialization/Version.java

```diff
@@ -635,48 +569,4 @@ public boolean isPre65() {
         .collect(Collectors.toList());
   }
 
-  public boolean isCurrentVersion() {
-    return this.ordinal == CURRENT.ordinal;
-  }
```

Comment:
this was used in exactly two places and is better accomplished via `Version.CURRENT.isEqual(someVersionOrdinal)`